packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

```diff
@@ -40,15 +40,15 @@ When a rate limit violation occurs, you're notified in the following ways and ca
 
 * You receive a rate limit email that informs you that you've hit a rate limit for an endpoint. Click **View utilization history** in the email to navigate to the rate limits dashboard and System Log for that particular endpoint.
 
-![Access the dashboard by email notification](/img/RLDashboard1.png "Displays the email notification of a rate limit violation with the link to the endpoint's rate limit dashboard")
+![Access the dashboard by email notification](/img/RLDashboard1.png)
```

Comment:
```suggestion
![Displays the email notification of a rate limit violation with the link to the endpoint's rate limit dashboard](/img/RLDashboard1.png "Access the dashboard by email notification")
```

Reply:
I'm not sure if you wanted to keep the image title or keep the alt-text? Based on the properties of the rendered image, the first string between [] is the alt-text, the second string in quotes is the image title.